src/clusterfuzz/_internal/bot/fuzzers/libFuzzer/engine.py

```diff
@@ -300,7 +300,8 @@ def fuzz(self, target_path, options, reproducers_dir, max_time):
     # If we exited with a non-zero return code with no crash file in output from
     # libFuzzer, this is most likely a startup crash. Use an empty testcase to
     # to store it as a crash.
-    if not crash_testcase_file_path and fuzz_result.return_code:
+    if (not crash_testcase_file_path and
+        fuzz_result.return_code not in constants.NONCRASH_RETURN_CODES):
       crash_testcase_file_path = self._create_empty_testcase_file(
```

Comment:
Let's also just change this to create file with less restrictive permissions. There's no reason to lock it down.